x/tunnel/types/packet.go

```diff
@@ -54,3 +54,26 @@ func (p Packet) GetContent() (PacketContentI, error) {
 
 	return packetContent, nil
 }
+
+func (p Packet) EncodeTss(
+	destinationChainID string,
+	destinationContractAddress string,
+	encoder feedstypes.Encoder,
+) ([]byte, error) {
+	encodingPacket, err := NewTssEncodingPacket(
+		p,
+		destinationChainID,
+		destinationContractAddress,
+		encoder,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	switch encoder {
+	case feedstypes.ENCODER_FIXED_POINT_ABI, feedstypes.ENCODER_TICK_ABI:
+		return encodingPacket.EncodeABI()
```

Comment:
in feeds, we use ABIEncode, think we should choose one.

Reply:
use EncodeABI in both feeds and tunnel modules